recipe/conda_build_config.yaml

```diff
@@ -1,9 +1,11 @@
+c_compiler:            # [win]
+- vs2019               # [win]
+cxx_compiler:          # [win]
+- vs2019               # [win]
+MACOSX_SDK_VERSION:    # [osx and x86_64]
+  - 10.12              # [osx and x86_64]
```

Comment:
Why is this needed?

Reply:
This was @xhochy's handiwork in https://github.com/conda-forge/llvmdev-feedstock/pull/152/commits/aeffbdc1a317d30e6a7db09d2bc1e83f5e2f9f0f / #143. Not sure what the consideration was at the time.

Reply:
There was some functionality that required a newer SDK but I don't know anymore what it was. We would need to get a failed build to see it.